mono/tests/Makefile.am

```diff
@@ -682,11 +682,14 @@ TESTS_CS_SRC=		\
 	bug-59400.cs \
 	tailcall-generic-cast-cs.cs \
 	tailcall-interface.cs \
-	sizeof-empty-structs.cs \
 	bug-60843.cs	\
-	roslyn-bug-19038.cs	\
 	nested_type_visibility.cs
 
+# some tests fail to compile on mcs
+if CSC_IS_ROSLYN
```

Comment:
You can use `ifeq ($(CSC),csc)`, that would avoid using another variable in `configure.ac`

Reply:
it's not that simple, `$(CSC)` can for example be `cd $srcdir && pwd`/external/roslyn-binaries/Microsoft.Net.Compilers/Microsoft.Net.Compilers.2.7.0/tools/csc.exe`. it's possible to fiddle around with some text functions, but it's kinda ugly

Reply:
IMO it would be easier to wrap the test inside of `#if !__MonoCS__` which is a define that is only set by the mcs compiler.

Reply:
@akoeplinger I disagree. In my understanding, `Makefile` serves as some sort of index about what tests are run in a given configuration. Your suggested solution would hide that information, and would require someone to know that a test can be disabled from _within_ a test. That could end up in a confusing state, as for example on CI, the test would still be showed as successfully executed.